Source/inv.cpp

```diff
@@ -435,17 +441,16 @@ bool CanEquip(const ItemStruct &item)
 /**
  * @brief A specialized version of 'CanEquip(int, ItemStruct&, int)' that specifically checks whether the item can be equipped
  * in one/both of the player's hands.
- * @param playerNumber The player number whose inventory will be checked for compatibility with the item.
+ * @param player The player whose inventory will be checked for compatibility with the item.
  * @param item The item to check.
  * @return 'True' if the player can currently equip the item in either one of his hands (i.e. the required hands are empty and
  * allow the item), and 'False' otherwise.
  */
-bool CanWield(int playerNumber, const ItemStruct &item)
+bool CanWield(PlayerStruct player, const ItemStruct &item)

```

Comment:
I believe this should be a ref parameter.
```suggestion
bool CanWield(PlayerStruct &player, const ItemStruct &item)
```